app/views/record/_record.html.erb

```diff
@@ -1,5 +1,5 @@
 <div id="full-record" class="gridband layout-3q1q wrap-full-record">
-  <div class="col3q box-content region full-record" data-region="Full record">
+  <main class="col3q box-content region full-record" data-region="Full record">
```

Comment:
I wonder what we're doing with the `data-region`. Just a curiosity and not a suggestion for a change.

Reply:
I don't know, and because of that I'm not comfortable changing it. Matt might have more info.

Reply:
Nothing is coming to mind for me about what the `data-region` value is doing here. Looking at the blame history for that line, it looks like it might have been brought over from Bento when _that_ system had a full record view:

https://github.com/MITLibraries/bento/blob/v1.12.5/app/views/record/record.html.erb#L9

Reply:
oooh right. This makes sense now. Bento likely had the bonus GA tracking enabled and may have tied into more detailed user interaction tracking.